docs/sources/reference/api/docker_remote_api_v1.12.md

```diff
@@ -115,7 +115,8 @@ Create a container
              "Memory":0,
              "MemorySwap":0,
              "CpuShares": 512,
-             "Cpuset": "0,1",
+             "CpusetCpus": "0,1",
```

Comment:
You are right, I think we should not change docs for previous version.
For now, change v1.16 and v1.17 is good enought.